src/private/WRLDBuildingApiHelpers.mm

```diff
@@ -0,0 +1,91 @@
+#include "WRLDBuildingApiHelpers.h"
+
+#import "WRLDBuildingHighlightOptions+Private.h"
+#import "WRLDBuildingDimensions+Private.h"
+#import "WRLDBuildingInformation+Private.h"
+#import "WRLDBuildingContour+Private.h"
+#import "WRLDMathApiHelpers.h"
+
+@interface WRLDBuildingApiHelpers ()
+
+@end
+
+@implementation WRLDBuildingApiHelpers
+{
+
+}
+
++ (Eegeo::BuildingHighlights::BuildingHighlightCreateParams)createBuildingHighlightCreateParams:(WRLDBuildingHighlightOptions*) buildingHighlightOptions
+{
+    CLLocationCoordinate2D selectionLocation = buildingHighlightOptions.selectionLocation;
+    Eegeo::Space::LatLong latLng = Eegeo::Space::LatLong::FromDegrees(selectionLocation.latitude, selectionLocation.longitude);
+
+    CGPoint selectionScreenPoint = buildingHighlightOptions.selectionScreenPoint;
+
+    int typeIntValue = static_cast<int>(buildingHighlightOptions.selectionMode);
```

Comment:
Static casting via int is potentially fragile? Possibly introduce adapter function to go direct from obj-c enum to C++ enum, similar to https://github.com/wrld3d/ios-api/blob/master/src/private/WRLDPositioner.mm#L22